configure.ac

```diff
@@ -899,14 +923,15 @@ done
 if test "$HAVE_CXXSTD" = TRUE; then
     AC_MSG_RESULT([yes ($CXXFLAGS_CXXSTD)])
 else
-    AC_MSG_ERROR(no)
+    AC_MSG_RESULT([no])
+    AC_MSG_ERROR(Compiler doesn't support C++17)
```

Comment:
True. But `AC_MSG_ERROR` prints 'error' before the message and that messes up the "checking ... " line. Instead of "checking whether gcc supports C++17... no" you get "checking whether gcc supports C++17... configure: error: no", which isn't as nice.

I know, it's minor thing, but now the message is consistent, followed with the last line giving the reason for failure: "configure: error: Compiler doesn't support C++17", which is easier to read/understand, IMHO.

Hope this makes sense.